distributed/utils.py

```diff
@@ -611,7 +611,7 @@ def read_block(f, offset, length, delimiter=None):
 
     Parameters
     ----------
-    fn: string
+    f: string
```

Comment:
In the code that follows, it is definitely a file (not a string), so `f`. I suspect that in a previous version, it was indeed a string, and this is vestigial.